Spec/PushActivationStateMachine.swift

```diff
@@ -91,41 +105,70 @@ override class var defaultTestSuite : XCTestSuite {
 
                     expect(stateMachine.current).to(beAKindOf(ARTPushActivationStateNotActivated.self))
                     expect(deactivatedCallbackCalled).to(beTrue())
+afterEach()
+
                 }
 
                 // RSH3a2
-                context("on Event CalledActivate") {
+                
+                    // RSH3a2a
+                    func test__Activation_state_machine__State_NotActivated__on_Event_CalledActivate__reusableTestsRsh3a2a(testCase: TestCase_ReusableTestsRsh3a2a) {
                     // RSH3a2a
-                    reusableTestsRsh3a2a()
+                    reusableTestsRsh3a2a(testCase: testCase, context: (beforeEach: beforeEach__Activation_state_machine__State_NotActivated, afterEach: afterEach))}
```

Comment:
Here's an example of non-nil `before/afterEach` hooks being passed to a `reusableTests…` function.

Reply:
Probably it could be just two additional parameters instead of one tuple. Just a suggestion.

Reply:
Yeah, that's a good suggestion. I can change that.

Reply:
I've made this change to the migrator.

Reply:
I've also made it so that these parameters have a default value of `nil`, which tidies things up at the point where we call these functions in a lot of cases.